torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -49,6 +49,11 @@ void copy_range(variable_list& out, IndexRange range, at::ArrayRef<Tensor> t) {
   std::copy(t.begin(), t.end(), out.begin() + range.first);
 }
 
+Tensor copysign_tensor_self_backward(const Tensor & grad, const Tensor & self, const Tensor & other) {
+  auto result = grad * self.sign() * (other.ge(0) * 2 - 1);
```

Comment:
I'm not sure this is correct. Consider the following:
```
c = torch.copysign(a, b)
   a,     b,     c,  grad
   1,  -0.0,  -1.0,  -1.0
```
Using the formula in the code we get the result is (1 * 1 * ((-0.0 > 0) * 2 - 1)) = (1 * 1 * ((1) * 2 - 1)) = 1, which is not what we wanted, right?